CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
Version 2.8 dates back to 2009. Do we need to support such old versions ?
https://cmake.org/pipermail/cmake/2009-November/033303.html

Consider using 3.2 (with GSL support), released in 2015: https://cmake.org/pipermail/cmake/2015-March/060082.html